jenkins/check-for-build.jenkinsfile

```diff
@@ -11,7 +11,8 @@ pipeline {
     }
     triggers {
         parameterizedCron '''
-            H 1 * * * %INPUT_MANIFEST=2.2.1/opensearch-2.2.1.yml;TARGET_JOB_NAME=distribution-build-opensearch
+            H/60 * * * * %INPUT_MANIFEST=2.2.1/opensearch-2.2.1.yml;TARGET_JOB_NAME=distribution-build-opensearch
```

Comment:
I think you can also add test job&manifest to the cron parameters here so it will also be triggered correctly.

Reply:
Hey @zelinh any example ?

Reply:
Example: https://github.com/opensearch-project/opensearch-build/pull/2273/files

Reply:
Thanks, just updated the change.